inky-build-pkg/action.yaml

```
@@ -43,3 +54,4 @@ runs:
         workspace-dir: ${{ github.workspace }}/build/${{ inputs.package-name }}
         empty-workspace: true
```

Should this be something like?
```suggestion
        empty-workspace: ${{ inputs.empty-workspace }}
```